dashboard/app/models/script.rb

```diff
@@ -209,6 +209,14 @@ def self.maker_unit_scripts
     visible_scripts.select {|s| s.family_name == 'csd6'}
   end
 
+  def self.text_to_speech_script_ids
+    Script.all.select(&:text_to_speech_enabled?).pluck(:id)
+  end
+
+  def self.pre_reader_script_ids
+    Script.all.select(&:pre_reader_tts_level?).pluck(:id)
+  end
```

Comment:
these seem to be why the homepage query count tests are failing, but it looks like this is a pattern we use, and the Scripts db table isn't very big, so I think this should be fine.

Reply:
@indhu-gunda Go ahead and update the query count on those failing tests and lets see if everything passes :)